### DISCUSSION TITLE
在发送20kb的图片时报错图片过大

### DISCUSSION THREAD

**Discussion by TDK1969** [Mrs4s/go-cqhttp#1393] [Q&A] [answered]
# 运行环境：
- ubuntu20.04
- nonebot2
- go-cqhttp：v1.0.0-rc1
# 问题描述
我想通过私聊发送图片，但是发现无论是使用url发送还是下载到本地再发送，都会报错如下：
> [2022-02-25 11:41:40] [WARNING]: 警告: 私聊 ********* 消息图片上传失败: file size over max 

相关代码如下：
``` python3
imgSend = on_command("imgSend", rule=to_me, aliases={"发图", "图片测试"})
@imgSend.handle()
async def AddBiliVideoFollow(matcher: Matcher, args: Message = CommandArg()):
    imgPath = '/root/project/NoneBot/TDK_Bot/src/file/biliStream/covers/haya.jpg'
    imgURI = pathlib.Path(imgPath).as_uri()
    size = os.path.getsize(imgPath) / 1024
    textMsg = '图片大小: {} kb'.format(size)
                    
    reportedMsg = Message([
        MessageSegment(type='text', data={'text':textMsg}),
        MessageSegment(type='image', data={'url':imgURI})
    ])

    await imgSend.finish(reportedMsg)
```
在私聊中收到消息如下：
![image](https://user-images.githubusercontent.com/54183084/155650404-465a02aa-e062-4a2d-a7df-a0dd4a2c3463.png)
仅收到文本，但没有收到图片

请问我该如何解决这个问题，诚心求助QAQ

**Comment by qianjunakasumi** [comment, marked as answer]
请尝试直接使用 go-cqhttp 相关接口复现，
若问题重现，请前往 Issues 中提交相关 BUG 说明
若问题无法得到复现（发送成功），请检查\咨询您基于 go-cqhttp 的框架有关方

**Reply by TDK1969** [reply]
已解决，查阅到了相关的文档
